samples/grpc/shopping-cart-service-scala/build.sbt

```diff
@@ -31,7 +31,7 @@ val AkkaHttpVersion = "10.5.0"
 val AkkaManagementVersion = "1.3.0"
 val AkkaPersistenceR2dbcVersion = "1.1.0-M8"
 val AkkaProjectionVersion =
-  sys.props.getOrElse("akka-projection.version", "1.4.0-M3-124-96b9d6c2-SNAPSHOT") // FIXME
+  sys.props.getOrElse("akka-projection.version", "1.4.0-M4") // FIXME
 val AkkaDiagnosticsVersion = "2.0.0"
 
```

Comment:
what about the RES samples?

Reply:
Missed those! Pushed now.

Reply:
Hmmm what. Pushed but github doesn't show it.